pyface/ui/qt4/font.py

```diff
@@ -0,0 +1,202 @@
+    """ Convert a Pyface font to a Qfont.
+
+    Parameters
+    ----------
+    font : pyface.font.Font
+        The Pyface font to convert.
+
+    Returns
+    -------
+    qt_font : QFont
+        The best matching Qt font.
+    """
+    qt_font = QFont()
+    families = []
+    default_family = None
+
+    for family in font.family:
+        if family not in generic_family_to_qt_family:
+            families.append(family)
+        elif default_family is None:
+            default_family = family
+
+    if families and hasattr(qt_font, 'setFamilies'):
+        # Qt 5.13 and later
+        qt_font.setFamilies(families)
+    elif families:
+        qt_font.setFamily(families[0])
+        # Note: possibily could use substitutions here,
+        # but not sure if global (which would be bad, so we don't)
+
+    if default_family is not None:
+        qt_font.setStyleHint(generic_family_to_qt_family[default_family])
+
+    qt_font.setPointSizeF(font.size)
+    qt_font.setWeight(weight_to_qt_weight[font.weight_])
+    qt_font.setStretch(font.stretch)
+    qt_font.setStyle(style_to_qt_style[font.style])
+    qt_font.setUnderline('underline' in font.variants)
+    qt_font.setStrikeOut('strikethrough' in font.variants)
+    qt_font.setOverline('overline' in font.variants)
+    if 'small-caps' in font.variants:
+        qt_font.setCapitalization(QFont.SmallCaps)
+    return qt_font
+
+
+def toolkit_font_to_properties(toolkit_font):
+    """ Convert a QFont to a dictionary of font properties.
+
+    Parameters
+    ----------
+    toolkit_font : QFont
+        The Qt QFont to convert.
+
+    Returns
+    -------
+    properties : dict
+        Font properties suitable for use in creating a Pyface Font.
+    """
+    family = []
+
+    if hasattr(toolkit_font, 'families'):
+        # Qt 5.13 and later
+        family = list(toolkit_font.families())
+    elif toolkit_font.family():
+        family.append(toolkit_font.family())
+    if toolkit_font.defaultFamily():
+        family.append(toolkit_font.defaultFamily())
+    family.append(qt_family_to_generic_family[toolkit_font.styleHint()])
+
+    size = toolkit_font.pointSizeF()
+    style = qt_style_to_style[toolkit_font.style()]
+    weight = map_to_nearest(toolkit_font.weight(), qt_weight_to_weight)
+    stretch = toolkit_font.stretch()
+    variants = set()
+    if toolkit_font.underline():
+        variants.add('underline')
+    if toolkit_font.strikeOut():
+        variants.add('strikethrough')
+    if toolkit_font.overline():
+        variants.add('overline')
+    if toolkit_font.capitalization() == QFont.SmallCaps:
+        variants.add('small-caps')
+
+    return {
+        'family': family,
+        'size': size,
+        'weight': weight,
+        'stretch': stretch,
+        'style': style,
+        'variants': variants,
+    }
+
+
+def map_to_nearest(target, mapping):
+    """ Given mapping with keys from 0 and 99, return closest value.
+
+    This is used to map back from
```

Comment:
Is this an incomplete sentence? Looks like it. "... Qt size to Pyface font size"